docs/dev_guide/deply_noobaa_on_minikube.md

```diff
@@ -0,0 +1,126 @@
+```bash
+make noobaa
+```
+Change the tag name  `noobaa:latest noobaa-core:<tag-name>`, for example: 
+```bash
+docker tag noobaa:latest noobaa-core:my-deploy
+```
+### 6) Deploy Noobaa (Noobaa-Operator Tab)
+```bash
+nb install --mini --noobaa-image='noobaa-core:my-deploy'
+```
+_Note: We have the alias to `nb` from the step 'Build Operator'._
+
+The installation should take 5-10 minutes.
+Once noobaa is installed please notice that the phase is Ready, you will see it in the CLI logs:
+
+✅ System Phase is "Ready".
+
+You can see something similar to this when getting the pods:
+```
+> kubectl get pods
+NAME                                               READY   STATUS    RESTARTS   AGE
+noobaa-core-0                                      1/1     Running   0          51m
+noobaa-db-pg-0                                     1/1     Running   0          51m
+noobaa-default-backing-store-noobaa-pod-a586c55b   1/1     Running   0          47m
+noobaa-endpoint-6cf5cccfc6-rmdrd                   1/1     Running   0          47m
+noobaa-operator-5c959d5564-qzgqb                   1/1     Running   0          51m
+```
+
+### 7) Wait For Default Backingstore to Be Ready (Noobaa-Operator Tab)
+We will use the default backingstore pod to run the tests, we need it to be in phase Ready, run:
+```bash
+kubectl wait --for=condition=available backingstore/noobaa-default-backing-store --timeout=6m
+```
+## Workarounds:
+### 1) MacOS:
+Change the `mini` resources inside file system.go (before building the operator image):
+
+```diff
+	if options.MiniEnv {
+		coreResourceList := corev1.ResourceList{
+-			corev1.ResourceCPU:    *resource.NewScaledQuantity(int64(100), resource.Milli),
+-			corev1.ResourceMemory: *resource.NewScaledQuantity(int64(1), resource.Giga),
++			corev1.ResourceCPU:    *resource.NewScaledQuantity(int64(500), resource.Milli),
++			corev1.ResourceMemory: *resource.NewScaledQuantity(int64(1), resource.Giga),
+		}
+		dbResourceList := corev1.ResourceList{
+-			corev1.ResourceCPU:    *resource.NewScaledQuantity(int64(100), resource.Milli),
+-			corev1.ResourceMemory: *resource.NewScaledQuantity(int64(500), resource.Mega),
++			corev1.ResourceCPU:    *resource.NewScaledQuantity(int64(800), resource.Milli),
++			corev1.ResourceMemory: *resource.NewScaledQuantity(int64(1), resource.Giga),
+		}
+		endpointResourceList := corev1.ResourceList{
+-			corev1.ResourceCPU:    *resource.NewScaledQuantity(int64(100), resource.Milli),
+-			corev1.ResourceMemory: *resource.NewScaledQuantity(int64(500), resource.Mega),
++			corev1.ResourceCPU:    *resource.NewScaledQuantity(int64(500), resource.Milli),
++			corev1.ResourceMemory: *resource.NewScaledQuantity(int64(1), resource.Giga),
+		}
+```
+
+### 2) Linux:
+Change te MakeFile and insert a comment in the lines (before building the core image):
+```diff
++ # DOCKER_BUILDKIT?=DOCKER_BUILDKIT=1
++ # ifeq ($(CONTAINER_ENGINE), podman)
++ # 	DOCKER_BUILDKIT=
++ # endif
+```
```

Comment:
Why?

Reply:
I took it from Linux users...
@alphaprinz @nadavMiz, Do you know?